include/deal.II/grid/grid_tools.h

```diff
@@ -2100,6 +2100,22 @@ namespace GridTools
   void
   partition_multigrid_levels(Triangulation<dim, spacedim> &triangulation);
 
+  /**
+   * Given cells identified by the current object, potentially living in a
+   * triangulation that is split among more than one processor, return which
+   * processor owns each one of them (i.e., what their "subdomain" is).
```

Comment:
I don't get the sentence. Isn't the next sentence enough?

Reply:
I removed this paragraph.